wagtailvideos/models.py

```diff
@@ -26,6 +25,8 @@
 from wagtail.wagtailsearch import index
 from wagtail.wagtailsearch.queryset import SearchableQuerySetMixin
 
+from enumchoicefield import ChoiceEnum, EnumChoiceField
```

Comment:
`isort` should not have moved this import down here, unless `enumchoicefield` was not installed in your virtualenv.